concepts/token.md

```diff
@@ -0,0 +1,11 @@
+---
+grand_parent: Resources
+parent: Concepts
+title: Token
+description: A short sequence of characters
+---
+
+A token is a short sequence of characters with meaning, most typically a word or punctuation character.
+Strings are split into tokens by [tokenisation](/customisation/tokenisation) algorithms.
+
+Examples: “this”, “sequence”, “?”
```

Comment:
@cefoo I think we should add something to the style guide that stuff like this should delimited `with backticks` so that later we can avoid translating it.

> Examples
> > `this`, `sequence`, `?`

And maybe in blockquote?

Reply:
@bittlingmayer

I would choose backticks.

We widely use blockquote in Events, Calls for papers, and in our Meetup 2 page, specifically for quotes, so I would leave that format for those cases.

I'll update the Style Guide.